test/libsolidity/SemanticTest.cpp

```diff
@@ -340,13 +379,31 @@ void SemanticTest::printUpdatedSettings(ostream& _stream, string const& _linePre
 
 void SemanticTest::parseExpectations(istream& _stream)
 {
-	TestFileParser parser{_stream};
+	TestFileParser parser{_stream, &m_builtins};
 	auto functionCalls = parser.parseFunctionCalls(m_lineOffset);
-	std::move(functionCalls.begin(), functionCalls.end(), back_inserter(m_tests));
+	move(functionCalls.begin(), functionCalls.end(), back_inserter(m_tests));
```

Comment:
Can we use `m_tests += parser.parseFunctionCalls(...)` here?

Reply:
nice! thats much better!